### PR TITLE
fix(sushiguard): inital v3

### PR DESCRIPTION
**Pull request by sambacha**

### CODE REVIEW

File: src/components/Settings/index.tsx [hunk 1]
```diff
@@ -10,11 +10,12 @@ import QuestionHelper from 'app/components/QuestionHelper'
...
+import { Feature } from 'app/enums'
+import { classNames, featureEnabled } from 'app/functions'
+import useWalletSupportsSushiGuard from 'app/hooks/useWalletSupportsSushiGuard'
 import { useActiveWeb3React } from 'app/services/web3'
 import { useToggleSettingsMenu } from 'app/state/application/hooks'
-import { useExpertModeManager, useUserOpenMev, useUserSingleHopOnly } from 'app/state/user/hooks'
+import { useExpertModeManager, useUserSingleHopOnly, useUserSushiGuard } from 'app/state/user/hooks'
 import React, { FC, useState } from 'react'
 
 import { OPENMEV_ENABLED, OPENMEV_SUPPORTED_NETWORKS } from '../../config/openmev'
```
Comment (chillichelli) [inline src/components/Settings/index.tsx:21]: Can we remove unused vars here?

File: src/lib/jsonrpc.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    // handle http errors (anything not 200)
+    if (res.status !== 200) throw new HttpJsonRpcError(res.statusText, { jsonrpc, id, method, params }, res)
+
+    // handle successful response
+    return res.json()
+  })
+}
+
+/** 
+ // TODO - add this as an Async Function
```
Comment (chillichelli) [inline src/lib/jsonrpc.ts:79]: Can this be deleted?

File: src/components/Settings/index.tsx (hunk 1 above)
Comment (sambacha) [reply src/components/Settings/index.tsx]: fixed in https://github.com/sushiswap/sushiswap-interface/pull/726

File: src/lib/jsonrpc.ts (hunk 2 above)
Comment (sambacha) [reply src/lib/jsonrpc.ts:79]: yup, fixed in https://github.com/sushiswap/sushiswap-interface/pull/726

**Comment by sambacha** [comment]
closing in favor of https://github.com/sushiswap/sushiswap-interface/pull/726

**Pull request closed without merging**